sections/extras.tex

```diff
@@ -95,6 +95,36 @@ \subsection{Finger pointing!?}
 
 \end{frame}
 
+\subsection{Fixup commits}
```

Comment:
I wouldn't put it into the extras. It's already so chaotic ... I prefer it in the "rewrite history" chapter and in any case, as discussed, rewrite this extras chapter.

Reply:
moved it between fixup and squash